### PR TITLE
Rename url field to uri in client payloads

### LINKED ISSUE #490: Use `uri` instead of `url` everywhere
Currently we mix `uri` and `url` for field names. As the core services uses `uri` we should use this everywhere.

Related: https://github.com/gnosis/safe-config-service/pull/160

### PR DESCRIPTION
**Pull request by fmrsabino**
Closes #490 

Given the current changes in the safe-config service https://github.com/gnosis/safe-config-service/pull/160, the `url` fields should be renamed to `uri`.

### CODE REVIEW

**Review by jpalvarezl — APPROVED** [review]
:+1:

**Comment by jpalvarezl** [comment]
There is still the methods in `src/config/mod.rs` named `base_config_service_url`, `base_exchange_api_url` and `redis_url` as well as their respective env variables: `CONFIG_SERVICE_URL`, `EXCHANGE_API_BASE_URL` and `REDIS_URL`. 

@rmeissner could we simply fork these into a separate issue, if we need to change them too?

**Review by jpalvarezl — APPROVED** [review]

**Review by jpalvarezl — CHANGES_REQUESTED** [review]
We are missing `SafeAppInfo::logo_url`

File: src/providers/info.rs [hunk 1]
```diff
@@ -172,7 +172,7 @@ impl<C: Cache> InfoProvider for DefaultInfoProvider<'_, C> {
             Ok(AddressEx {
```
Comment (jpalvarezl) [inline src/providers/info.rs:172]: :point_up: in line 65 `SafeAppInfo` has still a `logo_uri`

**Comment by rmeissner** [comment]
Yes, I would not adjust env variable names in this PR

**Comment by jpalvarezl** [comment]
For reference: https://github.com/gnosis/safe-client-gateway/issues/494

**Review by jpalvarezl — APPROVED** [review]

**Pull request merged**